### ISSUE TITLE
Double YouTube video playback issue. (specifically with 503 dev build of Firefox addon)

### ISSUE THREAD

**Issue by ApacheThunder** [YePpHa/YouTubeCenter#1724]
I have not tested the 503 build of the grease monkey script as I'm currently using 501. But The current dev build (dev build 503) there is a serious issue that forced me to disable the add-on.

About 90% of the time when a YouTube video loads it reloads it. (probably to get rid of ads. This part is normal). However a serious issue has cropped up. The video for whatever reason has two instances playing. One of which I can not see and can not interact with. I either have to kill plugin container or kill Firefox to get the second instance of the video (which is basically just audio playing since there is no where on the page I see it playing on) to stop playing. This might seem related to the ad blocking feature, but I recall this cropping up if I refresh the page my self so it's probably not directly related to that. Having a second instance of the video playing somewhere hidden where I could not stop playback forced me to completely disable the add-on. Until I see that it's fixed I'm staying on 501. My Firefox version is 37.0.2 and run Windows 7.

**Comment by Yonezpt** [comment]
Are you forcing the Flash Player on YouTube? Because yesterday they have made the HTML5 player by default for Firefox.

**Comment by ApacheThunder** [comment]
I believe so. I couldn't get the progress bar to stop auto hiding (I only want that happening in fullscreen), so I set it to Flash.

**Comment by Yonezpt** [comment]
Then that might be the issue, YTC is trying to force the flash player while YouTube is forcing the HTML5 player, resulting in the later ending up "ghosting" in the webpage out of sight.

Try resetting the YTC to default and then set the control bar to always visible. It's possible that there's an incorrect size you might have set up or a conflicting setting from previous versions, hopefully that will fix the always hiding bar.

As for the double player, it vwill have to wait for the author's attention.

**Comment by ApacheThunder** [comment]
I think I have size set to default. I don't really like having it centered. (just used to having the suggested videos go up all the way on the right side). But for now I'll stick to 501 version of greasemonkey script. There doesn't appear to be any issues I've had with it other then being unable to reply to comments without space bar pausing/playing video. But I fixed that by disabling shortcuts for the page. (but left them on for the player, so I can still use space to pause the video if it has focus)

**Comment by LaVey-Charkus-Veros** [comment]
I have the same problem A duplicate of the video plays on the background. However, for me it happens about 40% of the time, and reloading the page usually fixes it. I don't know if other add-ons (like AdBlock) help in making this problem less frequent.

I use "Aggressive flash" for player type at YTC's settings. Hopefully, this will be fixed in the next developer version (I'm using build 503).

**Comment by SINE** [comment]
I can agree on the problematic.

**Comment by ApacheThunder** [comment]
I just had the double playback issue crop up on build 501, so it's been around for a couple builds now. Doesn't seem to be as big a problem in 501 as it is in 503 though.

**Comment by LaVey-Charkus-Veros** [comment]
@ApacheThunder A few days ago, YouTube implemented a lot of changes. Most notably for YTC users, the video inside the player was pushed to the right; I believe that's why the developer hurried in releasing build 503 and give it a quick fix. Most likely, the double playback issue also originated that same day, so I guess we'll have to wait a few days more until the problem is identified and dealt with.

Regards.

**Comment by SINE** [comment]
Yes, another thing is that YouTube started forcing HTML5, as the others figured above, which will most likely be the cause of the problem.
